test/cases/create-document-fragment/create-document-fragment.spec.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * Use Case
+ * Run wcc against a custom element which creates a document fragment
```

Comment:
Sorry, I might have the lost recollection on this, but was there a particular need to test for document fragment specifically?  Not sure if the case / result maybe just need to cover a little bit more of the why, maybe that would help jog my recollection?